src/ConnectionMaker.cc

```diff
@@ -716,11 +717,13 @@ namespace ChimeraTK {
       accessor = _app.getTestableMode().decorate(accessor, detail::TestableMode::DecoratorType::READ);
     }
 
```

Comment:
Not overly happy with the external creation and passing on of the RecoveryHelper here, but also lacking a better idea

Reply:
I tried to look at decorateDeepInside but didn't really understand how/where to return {} to stop it from digging deeper

Reply:
Yes, you may not actually have the necessary control that way... It will go as deep in as the decorators on the "stack" allow.